tests/scikit_learn/etl/test_add_columns.py

```diff
@@ -20,20 +22,140 @@ def test_add_columns_success():
         }
     }
     instance = AddColumnsOperation(**arguments)
-    result = util.execute(instance.generate_code(), 
-                          dict([left_df, right_df]))
+    result = util.execute(instance.generate_code(),
+                          {'df1': left_df[1], 'df2': right_df[1]})
+
     assert result['out'].equals(util.iris(size=slice_size))
 
-# def test_add_columns_fail_different_row_number():
-#     assert False
-# 
-# def test_add_columns_fail_missing_parameters():
-#     assert False
-# 
-# def test_add_columns_fail_same_column_names():
-#     assert False
-# 
-# def test_add_columns_success_using_prefix():
-#     assert False
-# 
-# 
+
+def test_add_columns_fail_different_row_number():
+    """
+    In this case, AddColumnsOperation() creates a result
+    the size of the smallest slice_size.
```

Comment:
Yes. Maybe in future we could change it to have a parameter to define if uses the smallest data frame or if it uses the size of largest and fill incomplete data with null.